docs/security.html

```diff
@@ -44,8 +44,26 @@
             <li>keystore: the keystore file that stores the certificate. The keystore file contains the private key of the certificate; therefore, it needs to be kept safely.</li>
             <li>validity: the valid time of the certificate in days.</li>
         </ol>
-        Ensure that common name (CN) matches exactly with the fully qualified domain name (FQDN) of the server. The client compares the CN with the DNS domain name to ensure that it is indeed connecting to the desired server, not the malicious one.</li>
-
+        <br>
+	Note: By default the property ssl.endpoint.identification.algorithim has been set to null. Unless configured otherwise hostname verification will not performed. In order to enable hostname verification the following property will need to be set:
+
+	<pre>	ssl.endpoint.identification.algorithim = HTTPS </pre>
+
+	Once enabled clients will verify the server's fully qualified domain name (FQDN) against of the following two fields:
+	<ol>
+		<li>Common Name (CN)*
+		<li>Subject Alternative Name (SAN)
+	</ol>
+	<br>
+	*The Common Name, although technically valid, is only used to keep backward compatibility. In order to more strictly adhere to RFC-2818 the servers FQDN should be placed in a SAN field. To accomplish this append the following argument -ext SAN=DNS:{FQDN} to the keytool command:
```

Comment:
I think I would simply say "Both are valid, but SAN is recommended by RFC 2818". And then talk about how one can have multiple DNS entries under the SAN field (i.e. it's more flexible) leaving the CN for something more meaningful for authorization purposes.